conda_forge_tick/cli.xsh

```diff
@@ -31,7 +31,10 @@ def deploy(args):
             print(e)
 
     try:
-        load_graph()
+        gx = load_graph()
+        # TODO: be more selective about which json to check
+        for node, attrs in gx.nodes.items():
+            attrs._load()
```

Comment:
How much time does this take on your machine? I want to make sure it is not an hour or something?

Reply:
Also this fails on my machine. No attr _load.

Reply:
` attrs["payload"]._load()` works

it takes 5-10 seconds on my machine